go.mod

```diff
@@ -1,26 +1,27 @@
 module github.com/traefik/paerser
 
-go 1.19
+go 1.22.5
```

Comment:
```suggestion
go 1.22.0
```

Reply:
I've set 1.22.5 on purpose, because of the multiple security fixes included in this version.

Reply:
it's a library, you should never user patch version in this context

Reply:
Fixed with 5cfcdaa